generate.pl

```diff
@@ -222,3 +223,10 @@ =head1 DESCRIPTION
 WORKDIR /root
 
 CMD ["perl{{version}}","-de0"]
+
+FROM debian:{{_tag}}-slim as runtime
```

Comment:
As noted in https://github.com/docker-library/official-images/issues/3383#issuecomment-329259161 we end up with

- having to make builds at least twice per version/architecture
- a less-transparent dependency between the build base image and the runtime (unless we use the exact image layer ids here; @yosifkit @tianon can probably elaborate more)

I'm thinking of doing something similar to what https://github.com/travis-perl does (have prebuilts in a repo under here which we can fetch from the bashbrew builds) but I'm not sure if that's a correct (or supported) approach...

Reply:
I'm not really sure I follow all the comments (of #3383). As to your two remaining points

* The two builds should be fairly quick as they make use of the cache.
* You could also make a build-image based on debian:<tag>-slim.

Reply:
Yep, the second point is what we're exploring now in #52; I'll have a PR and perhaps a few demo images up on my own Docker Hub namespace for people to test.

Reply:
If you have something, I'd be happy to test this. Just let me know.

Reply:
Thanks!  I've just pushed #54, please try it out!  I've triggered builds for `5.28-slim` and `5.28-slim-threaded` in [my own hub repo](https://hub.docker.com/r/zakame/perl/tags/), though its taking a while so it might be better to build these locally :)